src/server/server.cc

```diff
@@ -117,6 +117,9 @@ Server::~Server() {
   }
 
   lua::DestroyState(lua_);
```

Comment:
Since `conn_ctxs_` is freed, should we remove or change the comment in L114?

Reply:
Also. should we adjust the ordering of `lua::DestroyState` and cleanup?( not so familiar with lua here, so don't know would we meet the problem)

Reply:
It should be fine to leave `lua::DestroyState` as it is because workers are stopped.

Reply:
> Since conn_ctxs_ is removed, should we remove or change the comment in L114?

Yes, deleted.